packages/components/src/validated-form-controls/control-with-error.tsx

```diff
@@ -38,6 +37,7 @@ function appendRequiredIndicator(
 	}
 	return label;
 }
+const VALIDITY_VISIBLE_ATTRIBUTE = 'data-validity-visible';
```

Comment:
I had trouble naming this. It's not "touched", because messages can be visible without the field being touched. It's also more like "visibility is activated", rather than "a message is visible right now".